package.json

```diff
@@ -30,7 +30,7 @@
     "generate:schema": "node ./scripts/generateSchemaTypes.js",
     "docz": "docz",
     "start": "NODE_ENV=production node dist/index.js",
-    "start:development": "NODE_ENV=development CONCURRENCY=2 TS_NODE_PROJECT=./src/tsconfig.json node -r ts-node/register -r tsconfig-paths/register ./src/index.ts",
+    "start:development": "NODE_ENV=development CONCURRENCY=${CONCURRENCY=2} TS_NODE_PROJECT=./src/tsconfig.json node -r ts-node/register -r tsconfig-paths/register ./src/index.ts",
```

Comment:
Bash Syntax: `${CONCURRENCY=2}` assigns the value 2 to  `CONCURRENCY`, better to make this clear using `${CONCURRENCY:=2}` (it's synonymous), but as we don't actually need or want to assign anything we can use the default and don't assign using `${CONCURRENCY:-2}`.